modules/ztp-pre-staging-booting-from-live-os.adoc

```diff
@@ -0,0 +1,126 @@
+${CONTAINER_TOOL} run --network host --rm -it quay.io/user/racadm:latest -r $IDRAC_IP -u root -p $PASSWD- serveraction powercycle
+----
+
+.Example output
++
+[source,terminal]
+----
+./rhcos-live-racadm.sh 10.19.28.53
+Security Alert: Certificate is invalid - self signed certificate
+Continuing execution. Use -S option for racadm to stop execution on certificate-related errors.
+Disable Remote File Started. Please check status using -s
+option to know Remote File Share is ENABLED or DISABLED.
+
+Security Alert: Certificate is invalid - self signed certificate
+Continuing execution. Use -S option for racadm to stop execution on certificate-related errors.
+Remote Image is now Configured
+
+Security Alert: Certificate is invalid - self signed certificate
+Continuing execution. Use -S option for racadm to stop execution on certificate-related errors.
+[Key=iDRAC.Embedded.1#VirtualMedia.1]
+Object value modified successfully
+
+Security Alert: Certificate is invalid - self signed certificate
+Continuing execution. Use -S option for racadm to stop execution on certificate-related errors.
+[Key=iDRAC.Embedded.1#ServerBoot.1]
+Object value modified successfully
+
+Security Alert: Certificate is invalid - self signed certificate
+Continuing execution. Use -S option for racadm to stop execution on certificate-related errors.
+Server power operation initiated successfully
+----
+
+[NOTE]
+====
+If you are targeting an HP server, you can also use the HP administration tools to manage HP iLOs.
+====
+
+. To interact with the Redfish interface, you can use the following commands:
+
+.. Check virtual media status:
++
+[source,terminal]
+----
+$ curl --globoff -H "Content-Type: application/json" -H \
+"Accept: application/json" -k -X GET --user ${username_password} \
+https://$BMC_ADDRESS/redfish/v1/Managers/Self/VirtualMedia/1 | python -m json.tool
+----
+
+.. Mount the ISO file as a virtual media:
++
+[source,terminal]
+----
+$ curl --globoff -L -w "%{http_code} %{url_effective}\\n" -ku ${username_password} -H "Content-Type: application/json" -H "Accept: application/json" -d '{"Image": "http://[$HTTPd_IP]/RHCOS-live.iso"}' -X POST https://$BMC_ADDRESS/redfish/v1/Managers/Self/VirtualMedia/1/Actions/VirtualMedia.InsertMedia
+----
+
+.. Set the boot order to boot from the virtual media once:
++
+[source,terminal]
+----
+$ curl --globoff  -L -w "%{http_code} %{url_effective}\\n"  -ku ${username_password}  -H "Content-Type: application/json" -H "Accept: application/json" -d '{"Boot":{ "BootSourceOverrideEnabled": "Once", "BootSourceOverrideTarget": "Cd", "BootSourceOverrideMode": "UEFI"}}' -X PATCH https://$BMC_ADDRESS/redfish/v1/Systems/Self
+----
+
+. Reboot and ensure that the server is booting from virtual media.
+
+
+It is recommended to permit logging into the server through SSH before partitioning. To give login permissions, do the following:
```

Comment:
Lately, we have found a better way to enable SSHD. It is basically building a custom RHCOS Image using coreos supported utilities that come with SSHD enabled. I think it is worth mentioning the new method because the previous one shown here is not automated or agile.

Reply:
@alosadagrande Do you have more details on this? Or do you think it would be enough to just mention what you wrote here?